libse/Forms/SplitLongLinesHelper.cs

```diff
@@ -8,25 +8,25 @@ public static class SplitLongLinesHelper
         public static bool QualifiesForSplit(string text, int singleLineMaxCharacters, int totalLineMaxCharacters)
         {
             string s = HtmlUtil.RemoveHtmlTags(text.Trim(), true);
+            // TODO: Exclude \r\n.
             if (s.Length > totalLineMaxCharacters)
                 return true;
 
-            var arr = s.SplitToLines();
-            foreach (string line in arr)
+            foreach (string line in s.SplitToLines())
             {
                 if (line.Length > singleLineMaxCharacters)
                     return true;
             }
 
-            var tempText = Utilities.UnbreakLine(text);
```

Comment:
tempText will still contain html tags if present which will give us unwanted result in line 31